Crate does not build due to stdarch_arm_crc32 feature

[`stdarch_arm_crc32`](https://github.com/rust-lang/stdarch/pull/1573) is now stable and use of the feature flag will fail builds:

```
error[E0635]: unknown feature `stdarch_arm_crc32`
  --> /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/pathfinder_simd-0.5.3/src/lib.rs:13:69
   |
13 | #![cfg_attr(all(target_arch = "aarch64", pf_rustc_nightly), feature(stdarch_arm_crc32))]
```

--- a/simd/src/lib.rs
+++ b/simd/src/lib.rs
@@ -10,7 +10,6 @@
 
 #![cfg_attr(pf_rustc_nightly, allow(internal_features))]
 #![cfg_attr(pf_rustc_nightly, feature(link_llvm_intrinsics, core_intrinsics))]
-#![cfg_attr(all(target_arch = "aarch64", pf_rustc_nightly), feature(stdarch_arm_crc32))]
 #![cfg_attr(pf_rustc_nightly, feature(simd_ffi))]
 
 //! A minimal SIMD abstraction, usable outside of Pathfinder.

